### ISSUE TITLE
Keyboard focus issues

### ISSUE THREAD

**Issue by msrichardson** [conveyal/trimet-mod-otp#138]
The heuristic analysis identified keyboard focus issues, particularly in regards to tabbing through the application.  For example, unable to focus on the date/time and mode selector edit buttons.  It was recommended that we "ensure keyboard focus is present and ordered in a logical way."
